Trivy local setup document update (#24)
UKHO/devops-pipelinetemplates: trivy/README.md

Trivy will generate and publish results in the build pipeline under the Test and Coverage sections, allowing you to access a comprehensive overview of successes and failures. Utilise the filter to seamlessly switch between the two views. Furthermore, any failed scans will be clearly presented within the Docker console for immediate visibility and action.

> ## local setup

Reviewer: ```suggestion
## Local setup
```

> ## local setup

Reviewer 2: ```suggestion
## Local setup
```

> Trivy can be installed locally by downloading the windows compatible binaries [from](https://github.com/aquasecurity/trivy/releases/latest/) and updating binary files location at `PATH` system environment variable.

Reviewer 2: ```suggestion
Trivy can be installed locally on Windows by downloading the [Windows compatible binaries](https://github.com/aquasecurity/trivy/releases/latest/) and adding the location of the files to the Windows PATH environment variable.
```

> Running trivy scan at command prompt locally as below:

Reviewer 2: ```suggestion

Once installed, run a Trivy scan using a command prompt be navigating to your Trivy location and running commands as below:
```

- To scan a local project including language-specific files:
`trivy fs /path/to/your_project_folder`

- To scan a single file:
`trivy fs ./trivy-ci-test/single_file`,

> Other operating systems installation instructions can be found [here](https://aquasecurity.github.io/trivy/v0.18.3/installation/)

Reviewer 2: ```suggestion
Other operating system installation instructions can be found [here].(https://aquasecurity.github.io/trivy/v0.18.3/installation/)
```

> Instructions to use Trivy in [VS Code plugin](https://github.com/aquasecurity/trivy-vscode-extension)

Reviewer 2: ```suggestion
A [VS Code plugin](https://github.com/aquasecurity/trivy-vscode-extension) for Trivy is available.
```

Trivy will generate and publish results in the build pipeline under the Test and Coverage sections, allowing you to access a comprehensive overview of successes and failures. Utilise the filter to seamlessly switch between the two views. Furthermore, any failed scans will be clearly presented within the Docker console for immediate visibility and action.

## Local setup

Trivy can be installed locally on Windows by downloading the [Windows compatible binaries](https://github.com/aquasecurity/trivy/releases/latest/) and adding the location of the files to the Windows PATH environment variable.

> Once installed, run a Trivy scan using a command prompt be navigating to your Trivy location and running commands as below:

Reviewer 2: ```suggestion
Once installed, run a Trivy scan using a command prompt by navigating to your Trivy location and running commands as follows:
```

- To scan a local project including language-specific files:
`trivy fs /path/to/your_project_folder`

- To scan a single file:
`trivy fs ./trivy-ci-test/single_file`,

> Other operating system installation instructions can be found [here].(https://aquasecurity.github.io/trivy/v0.18.3/installation/)

Reviewer 2: ```suggestion
Other operating system installation instructions can be found [here](https://aquasecurity.github.io/trivy/v0.18.3/installation/).
```
